aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,63 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
 
-.. note::
+ The follwoing instrcution were tested using CentOS 7.
+ 
+Install Mono
```

Comment:
I suggest following the same structure that we now have for the Ubuntu section. We want the two sections to be consistent where they can be consistent.